.ci/build-freebsd.sh

```diff
@@ -5,20 +5,6 @@
 # shellcheck disable=SC2046
 git submodule -q update --init --depth 1 $(awk '/path/ && !/llvm/ && !/opencv/ { print $3 }' .gitmodules)
 
-# Prefer newer Clang than in base system (see also .ci/install-freebsd.sh)
-# libc++ isn't in llvm* packages, so download manually
-fetch https://github.com/llvm/llvm-project/releases/download/llvmorg-19.1.7/llvm-project-19.1.7.src.tar.xz
-tar xf llvm*.tar.xz
-export CC=clang19 CXX=clang++19
```

Comment:
Will it use clang without this?

Reply:
Yep. FreeBSD uses Clang/libc++ by default since 10.0, released on 2014-01-20.

13.4 ships Clang/libc++ 18 while upcoming FreeBSD [13.5](https://www.freebsd.org/releases/13.5R/schedule/) should have Clang/libc++ 19. Unless RPCS3 switches to C++23 before that it's unlikely to cause issues.